Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:5
+
+ENV PHANTOMJS_VERSION 1.9.7
+
+# https://hub.docker.com/r/cmfatih/phantomjs/~/dockerfile/
+RUN \
+  mkdir -p /srv/var && \
+  wget -q --no-check-certificate -O /tmp/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 && \
+  tar -xjf /tmp/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 -C /tmp && \
+  rm -f /tmp/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 && \
+  mv /tmp/phantomjs-$PHANTOMJS_VERSION-linux-x86_64/ /srv/var/phantomjs && \
+  ln -s /srv/var/phantomjs/bin/phantomjs /usr/bin/phantomjs
+
+COPY package.json /checklistomania/
+
+WORKDIR /checklistomania
+
+RUN npm install -g bower
+RUN npm install
```

Comment:
Hmmmm will this actually do what we want it to do?  I haven't tried this PR on my system but my guess is that it'd leave you with an empty `/checklistomania/node_modules` directory when running your `app` container... That's part of why in [CALC's `Dockerfile`](https://github.com/18F/calc/blob/develop/Dockerfile) we actually run the `npm install` in a `/node` directory in CALC, and then add `/node/node_modules` to `NODE_PATH`... Oy node's way of doing things makes dockerizing it so complicated and confusing.